model/host/static_hosts.go

```diff
@@ -35,16 +35,13 @@ func MarkInactiveStaticHosts(activeStaticHosts []string, d *distro.Distro) error
 		}
 	}
 
-	toTerminate, err := Find(db.Query(query))
-	if adb.ResultsNotFound(err) {
```

Comment:
I removed it because I think Find (the [old version](https://github.com/evergreen-ci/evergreen/blob/e1789d96a1729ae60ad9dd48d6401bb9a2f16cd5/db/query.go#L103) and the new) will never return this error.